assets/scss/templates/_guidelines.scss

```diff
@@ -209,3 +209,23 @@ body {
     padding: 5px 0;
   }
 }
+
+
+/* tooltip */
+
+.wrapper-tooltip {
+  flex-wrap: wrap;
+  display: flex;
+  justify-content: space-between;
+  padding: 100px 10px;
+}
+
+.dp-box {
+  /* We can use the following parameters to customize them */
+  /* Mixin parameters $color - $width - $max-width - $min-height */
```

Comment:
I think that the comments are not necessary anymore, what do you think @fcoronelmakingsense ?